pycbc/workflow/splittable.py

```diff
@@ -147,8 +150,8 @@ def setup_splittable_dax_generated(workflow, input_tables, out_dir, tags):
             num_injs = int(cp.get_opt_tags("em_bright_filter", "max-keep",
                                            tags))
         else:
-            num_injs = int(cp.get_opt_tags("workflow-injections", "num-injs",
-                                           tags))
+            num_injs = int(cp.get_opt_tags("workflow-injections",
+                                           "ninjections", tags))
```

Comment:
This seems to require a syntax change in the configuration file. It would be good if the new workflow could run with old configuration files as much as possible. Given that this is a trivial change, can we avoid it for now?

Reply:
I remembered why I did this... it was because `pycbc_create_injections` uses this variable name. I've reverted it and a comment clarifying this reasoning. I can also add a check for the injections executable, or there could be a better way to handle it.

Reply:
Ok.